automation/test.sh

```diff
@@ -325,5 +325,7 @@ EOF
 fi
 
 
+ginko_params="$ginko_params --ginkgo.focus=ISCSI"
```

Comment:
You could just do `FUNC_TEST_ARGS=" --ginkgo.focus=ISCSI" make functest ` when invoking the tests instead of focusing here, not sure you knew.

Reply:
I changed tests.sh because it is the use that run on CI, on my local host I use what you mentioned :)